tests/dep_test_permutation.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Testing File
+"""
+# Import necessary modules
+import numpy as np
+import pyrft as pr
+import os
+
+# The location where the pyrft module is saved (need to specify yourself)
+pyrft_loc = 'C:/Users/12SDa/davenpor/davenpor/Toolboxes/pyrft/'

```

Comment:
This is forbidden. Cleanly import your module.

Reply:
This file is deprecated - as denoted by the dep suffix.

Reply:
But yes I'll keep that in mind

Reply:
Don't keep deprecated file, please remove them.

Reply:
Okay removed!